editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2768,7 +2825,7 @@ void SpatialEditorViewport::set_can_preview(Camera *p_preview) {
 
 	preview = p_preview;
 
-	if (!preview_camera->is_pressed())
+	if (!preview_camera->is_pressed() && !previewingCinema)
```

Comment:
It's just a nitpick but we use `snake_case` for identifiers, so `previewing_cinema` would be better.